PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
cfi files are supposed to have the same name as the (only) instance of a module defined inside